flutter_local_notifications/README.md

```diff
@@ -115,12 +115,13 @@ If your application needs the ability to schedule notifications then you need to
 <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
 ```
 
-The following is also needed to ensure scheduled notifications remain scheduled upon a reboot (this is handled by the plugin)
+The following is also needed to ensure scheduled notifications remain scheduled upon a reboot, and after the app was updated (this is handled by the plugin)
```

Comment:
Just realised I had some awkward wording around this. Would you mind rewording this to 

_The following is also needed to ensure  notifications remain scheduled upon a reboot and after an application is updated_